### DISCUSSION TITLE
How can we help

### DISCUSSION THREAD

**Discussion by r9mp** [nuxt/nuxt#18347] [General]
Hey guys,

Thanks for this new release and all the great work accomplished! :) 
Hope your all going well, next weeks will be crazy 🥲 

As issues / discussions seems to be going nuts, I wanted to create this post 🆘. Maybe create your own and pin it... Do what suits you best.
Hope this post is not a duplicate!

I create this issue, because I want to help on the documentation but I'm not sure how to, as it seems that there is still pages missing (for example: plugins (empty), modules (hidden), ways to deploy, fetchState dissapeared, ...). => I'm a bit lost in how to help 'cause I don't see the difference between what's missing, what's in progress (on your side) and what's changed. I am checking the source code (your code & the docs code), but I'm not sure it's the best / quickiest way to help. And there is a risk of duplicating development.

So guys, what can I/we do? 😄

@all feel free to comment here if you want to help in other ways. I'll edit my post to summarise everything.

**Comment by lucpotage** [comment]
On my side, I try to answer as many questions as possible in Discussions so that Nuxt makers can focus on the rest. It helps me discover Nuxt 3 at the same time.
